scania-digital-design-system/sdds
# docs(side-menu): made container full height to demonstrate side-menu correctly

**Describe pull-request**  
Made container full height to demonstrate side-menu correctly.

**Solving issue**  
Fixes: AB#2730

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Side menu
3. Check the story.

**Screenshots**  
-

**Additional context**  
-
